packages/ChangesReloaded-Core.package/CRHistoryListItem.class/instance/ignore.st

```diff
@@ -0,0 +1,5 @@
+selecting
+ignore
+
+	self history do: [:change |
+		change isSelected: false]
```

Comment:
I think the line wouldn't be too long.
```suggestion
	self history do: [:change | change isSelected: false]
```

Reply:
do we have a rule for that one? I think we are doing it kinda too arbitrary right now :/

Reply:
I'd be for the rule: New context, new line (it's code in a new block, so make that clear with a new line). Don't have too strong feelings about it tho